medusa/server/api/v1/core.py

```diff
@@ -163,25 +168,25 @@ def call_dispatcher(self, args, kwargs):  # pylint:disable=too-many-branches
             or calls the TVDBShorthandWrapper when the first args element is a number
             or returns an error that there is no such cmd
         """
-        logger.log(u"API :: all args: '" + str(args) + "'", logger.DEBUG)
-        logger.log(u"API :: all kwargs: '" + str(kwargs) + "'", logger.DEBUG)
+        log.debug(u'API :: all args: {0!r}', args)
+        log.debug(u'API :: all kwargs: {0!r}', kwargs)
```

Comment:
Okay. I still think repr should be used for object representation. If we really need to have the single quotes (do we need them anyway?), we can use double quotes for the outer quotes.